packages/generator/__tests__/create-anatomy.ts

```diff
@@ -0,0 +1,54 @@
+// https://github.com/chakra-ui/zag/blob/215df8c66e953de02904c8419c1650f344e28faa/packages/anatomy/src/create-anatomy.ts
```

Comment:
Why not install zag-js anatomy for this one?

Reply:
cause it's just for tests and I didnt want to have the full selectors

https://github.com/chakra-ui/panda/pull/1544/files/ccf4825779a2a6855772a81370cadc496dcc4603..b5d8245cc937744aadcaa6c9a2ceb222ca52bd9b#diff-c836854eabc759fe5cdbff040b25cef0db437818faf43c9ad507b5294546432dR35-R39